source/ShopPanel.cpp

```diff
@@ -473,6 +486,17 @@ bool ShopPanel::Click(int x, int y, int /* clicks */)
 			return Scroll(0, -4);
 	}
 
+	auto ScrollbarClick = [x, y, clicks](auto &scrollbar, auto &scroll) {
+		return scrollbarMaybeUpdate([x, y, clicks](auto &scrollbar) {
+			return scrollbar.Click(x, y, clicks);
+		}, scrollbar, scroll, true);
+	};
+	if(ScrollbarClick(mainScrollbar, mainScroll)
+		|| ScrollbarClick(sidebarScrollbar, sidebarScroll)
+		|| ScrollbarClick(sidebarScrollbar, sidebarScroll))
```

Comment:
```suggestion
			|| ScrollbarClick(sidebarScrollbar, sidebarScroll)
			|| ScrollbarClick(sidebarScrollbar, sidebarScroll))
```
double indent looks better if there are no brackets